packages/binding-coap/src/coap-server.ts

```diff
@@ -164,6 +164,26 @@ export default class CoapServer implements ProtocolServer {
     });
   }
 
+  public destroy(thingId: string): Promise<boolean> {
+    console.debug("[binding-coap]", `CoapServer on port ${this.getPort()} destroying thingId '${thingId}'`);
+    return new Promise<boolean>((resolve, reject) => {
+      let removedThing: ExposedThing = undefined;
+      for (let name of Array.from(this.things.keys())) {
+        let expThing = this.things.get(name);
+        if (expThing != null && expThing.id != null && expThing.id === thingId) {
```

Comment:
I think you can compact this into:
`if(expThing?.id === thingId)`

Reply:
+1 makes sense (old school developer ;-))